interface/src/app/gene/searchMany/searchMany.js

```diff
@@ -11,22 +11,34 @@ angular.module('adage.gene.searchMany', [
       views: {
         'main': {
           templateUrl: 'gene/gene-network.tpl.html',
-          controller: ['$scope', 'UserFactory',
-            function($scope, UserFactory) {
-              $scope.userObj = null;
-              UserFactory.getPromise().$promise.then(function() {
-                $scope.userObj = UserFactory.getUser();
-              });
-
-              // TODO: Right now, we are hard-coding this organism
-              // as Pseudomonas (since it is the only one currently
-              // supported by ADAGE). However, as we incorporate
-              // multi-species support, this organism will have to
-              // be obtained from the ML model. This is the same as
-              // the issue in geneSearchForm (also with $scope.organism).
-              $scope.organism = 'Pseudomonas aeruginosa';
-            }
-          ]
+          controller: ['UserFactory', function(UserFactory) {
+            var self = this;
+            self.userObj = null;
+            UserFactory.getPromise().$promise.then(function() {
+              self.userObj = UserFactory.getUser();
+            });
+
+            // TODO: Right now, we are hard-coding this organism
+            // as Pseudomonas (since it is the only one currently
+            // supported by ADAGE). However, as we incorporate
+            // multi-species support, this organism will have to
+            // be obtained from the ML model. This is the same as
+            // the issue in geneSearchForm (also with $scope.organism).
+            self.organism = 'Pseudomonas aeruginosa';
+
+            // '$scope.autocomplete' holds a boolean value, of whether or
+            // not the user wants to use autocomplete search to llok up a
```

Comment:
Typo: "llok" should be "look".
About the bug, one interesting thing is that if you use **ng-show** or **ng-hide** instead of **ng-if**, you won't have this problem, because ng-show/ng-hide doesn't generate a new scope.